doc/api/cluster.md

```diff
@@ -626,6 +626,8 @@ values are `"rr"` and `"none"`.
     (Default=`process.argv.slice(2)`)
   * `silent` {Boolean} whether or not to send output to parent's stdio.
     (Default=`false`)
+  * `stdio` {Array} Configures the stdio of forked processes. When this option
+    is provided, it overrides `silent`.
```

Comment:
hmm... perhaps the implementation should simply override or error if `ipc` is not passed. If overridden, a warning can be emitted.